_schemaV2.graphql

```diff
@@ -3639,6 +3639,17 @@ type CitySponsoredContent {
   ): ShowConnection
 }
 
+input CollectArtistsMutationInput {
+  artistIDs: [String!]!
+  clientMutationId: String
+}
+
+type CollectArtistsMutationPayload {
```

Comment:
How come this isn't a union with the error type, and so forth - we should be using the standard `orError` pattern for all mutations.